src/sagemaker_containers/serializers.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json as stdjson
+import textwrap
+
+import numpy as np
+from six import BytesIO, StringIO
+
+from sagemaker_containers import content_types
+
+
+class Npy(object):
+    """Npy Serializer"""
+
+    @staticmethod
+    def loads(obj):  # type: (object) -> object
+        """Load arrays or pickled objects from memory into numpy arrays.
+
+        Args:
+            obj (object): object to be deserialized.
+
+        Returns:
+            (np.array): deserialized numpy array.
+        """
+        stream = BytesIO(obj)
+        return np.load(stream)
+
+    @staticmethod
+    def dumps(nparray):  # type: (np.array) -> object
+        """Serializes an numpy array into memory.
+
+        Args:
+            nparray (np.array): np array object to be serialized.
+
+        Returns:
+            (obj): serialized np array.
+        """
+        buffer = BytesIO()
+        np.save(buffer, nparray)
+        return buffer.getvalue()
+
+
+class Json(object):
```

Comment:
JsonSerializer. Soo many things called json or Json already.

Reply:
JsonSerializer is not Pythonic. The module (namespace) solves that problem for you.

from sagemaker_container.serializers import Json

Is not approved by PEP 8, Goggle style guide, and fails on linters.

Reply:
I disagree that the namespace solves this, because after you import, you end up with a bunch of similarly named things.

how would ...serializers.JsonSerializer violate PEP8 or fail a lint check?

Reply:
```from sagemaker_container.serializers import Json``` is not approved:

https://google.github.io/styleguide/pyguide.html#Imports

Use imports for packages and modules only.

Reply:
yes, it should be:

```
import sagemaker_container.serializers 
...
# at point of use
serializer = sagemaker_container.serializers.JsonSerializer()
```
if a user (not us) finds that "inconveniently long", they can add an alias

```
import sagemaker_container.serializers as ser # (or whatever)

serializer = ser.JsonSerializer()
```